demos/apps/echo-with-auth/echo-app.yaml

```diff
@@ -0,0 +1,45 @@
+# Example service with an injected proxy
+apiVersion: v1
+kind: Service
+metadata:
+  name: echo
+  labels:
+    app: echo
+spec:
+  ports:
+  - port: 80
+    targetPort: 80
+    name: http
+  - port: 8080
+    targetPort: 8080
+    name: http-alternative
+  selector:
+    app: echo
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: echo
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: echo
+        version: unversioned
```

Comment:
remove label. I would suggest removing any extraneous data that might confuse the end user